orchestrator/src/main/java/com/yahoo/vespa/orchestrator/resources/InstanceResource.java

```diff
@@ -73,4 +76,32 @@ public InstanceStatusResponse getInstance(@PathParam("instanceId") String instan
         Map<HostName, String> hostStatusStringMap = OrchestratorUtil.mapValues(hostStatusMap, HostStatus::name);
         return InstanceStatusResponse.create(applicationInstance, hostStatusStringMap);
     }
+
+    @GET
+    @Path("/{instanceId}/slobrok")
+    @Produces(MediaType.APPLICATION_JSON)
+    public List<SlobrokEntryResponse> getSlobrokEntries(
+            @PathParam("instanceId") String instanceId,
+            @QueryParam("pattern") String pattern) {
+        ApplicationInstanceReference reference = parseInstanceId(instanceId);
+        ApplicationId applicationId = OrchestratorUtil.toApplicationId(reference);
+
+        if (pattern == null) {
```

Comment:
Consider using `@DefaultValue` on query param.

Reply:
If you don't mind I'd keep this as-is, but I'll add a test that verifies null is handled to produce "**" pattern. The test wouldn't make sense with @DefaultValue.